pages/nft/iscn/_iscnId.vue

```diff
@@ -798,39 +845,52 @@ export default class NFTTestMintPage extends Vue {
 
   async createNftClass() {
     await this.initIfNecessary()
-    if (!this.signer) return
+    if (!this.signer) return '';
```

Comment:
Should this guard be put in the else below like L960?

Reply:
yes indeed